src/abstract-command.js

```diff
@@ -297,6 +303,111 @@ class AbstractCommand {
     this.logger.warn('THUB_TOKEN is not provided.');
     return Promise.resolve();
   }
+
+
+  /**
+   *   ******************* NEW *******************
+   */
+
+  /**
+   * @param {Object} fullConfig
+   * @returns {Promise}
+   */
+  checkProjectDuplicateComponents(fullConfig) {
```

Comment:
check if this function is used in `DistributedCommand` class

Reply:
transferred to DistributedCommand